README.rst

```diff
@@ -180,14 +180,15 @@ https://github.com/glenjarvis/ansible_tutorial/issues/78
 3. Activate the Virtual Enviroment. Every time you come back to work on this
    project, you will need to activate your virtual environment::
 
-       $ cd full_path_to_this_repo
+       $ cd [full_path_to_this_repo]
```

Comment:
I like what you are doing here. Do you mind using the more traditional <full path to this repo>. The change that I suggest is:

`$cd <full path to this repo>`

Without the spaces makes it more clear to me that this isn't an actual command. What do you think?